packages/blade/src/components/Box/getBoxStyles.ts

```diff
@@ -4,14 +4,17 @@ import type { BoxProps } from './Box';
 import { getIn, makeSize, makeSpace } from '~utils';
 
 // allows 'auto' as spacing value
-const getAutoSpacingValue = <T extends string | undefined>({
+const getSpacingValue = <T extends string | number | undefined>({
```

Comment:
Why are we allowing numbers? Won't that break the purpose of tokens 🤔 
If there's a legit usecase we should add a comment to explain the intent

Reply:
+1. Now that we don't need all the "absolute" hacks. Can we get avoid allowing free-form margins?

Reply:
will fix this in Pr-3 since the everything resides there for this so will check